miner/scroll_worker.go

```diff
@@ -658,6 +659,10 @@ func (w *worker) processTxnSlice(txns types.Transactions) (bool, error) {
 // processReorgedTxns
 func (w *worker) processReorgedTxns(reason error) (bool, error) {
 	reorgedBlock := w.chain.GetBlockByNumber(w.current.header.Number.Uint64())
+	if reorgedBlock == nil {
```

Comment:
is this due to "if there is a tx that will be evicted in n+1's reorg block, it will cause panic"?

Reply:
It is due to this function being called for `n+1`th block now. Which is the last block we create to trigger the reorg that doesn't have a canonical counterpart. So `GetBlockByNumber(n+1)` returns nil.

Reply:
got it. oh yes, no need to actually have "to be skipped" tx.